src/dune_rules/coq_config.ml

```diff
@@ -0,0 +1,16 @@
+open Import
+open Memo.O
+
+let impl bin =
+  let* _ = Build_system.build_file bin in
+  let+ line =
+    Memo.of_non_reproducible_fiber
```

Comment:
Does this mean that even within a single run we will re-exec?

All Coq config is deterministic, just set a Coq configure time.

Reply:
And guarantee to change only if the hash changes.